tests/test_composition.py

```diff
@@ -1180,12 +1234,38 @@ def test_deeply_nested(self):
         self.assertEqual(stack.transformed_time(fifty, clip), middle)
         self.assertEqual(stack.transformed_time(ninetynine, clip), last)
 
+        def _nest(self, item):
```

Comment:
Should this just be a member method of the test class?  that way you wouldn't have to explicitly pass in `self` below.  Could stay `_` prefixed.